server/resource.go

```diff
@@ -274,6 +274,10 @@ func (s *Server) registerResourcePublicRoutes(g *echo.Group) {
 		}
 		c.Response().Writer.Header().Set(echo.HeaderCacheControl, "max-age=31536000, immutable")
 		c.Response().Writer.Header().Set(echo.HeaderContentSecurityPolicy, "default-src 'self'")
+		if strings.HasPrefix(resourceType, "video") || strings.HasPrefix(resourceType, "audio") {
+			http.ServeContent(c.Response(), c.Request(), resource.Filename, time.Unix(resource.UpdatedTs, 0), bytes.NewReader(resource.Blob))
+			return nil
+		}
 		return c.Stream(http.StatusOK, resourceType, bytes.NewReader(resource.Blob))
```

Comment:
Good catch! I think we can use `c.Blob` instead of `c.Stream`. Could you take a test for it in your machine again
https://echo.labstack.com/guide/response/#send-blob

Reply:
Actually I tested it before and `c.Blob` doesn't work .. 
<img width="796" alt="image" src="https://user-images.githubusercontent.com/16509323/213646937-25c51644-b4eb-410c-8cec-190d0b3e2418.png">
And it also got `Failed to Load Resource, Plugin Handled Load` on safari 

Test code:
```golang
f, _ := os.Open("./video.mp4")
fd, _ := io.ReadAll(f)

// Routes
e.GET("/test-httpserve.mp4", func(c echo.Context) error {
	ff := bytes.NewReader(fd)
	http.ServeContent(c.Response(), c.Request(), "video.mp4", time.Now(), ff)
	return nil
})
e.GET("/test-blob.mp4", func(c echo.Context) error {
	return c.Blob(http.StatusOK, "video/mp4", fd) // also try http.StatusPartialContent and not work..
})
```